www/docs/en/dev/config_ref/index.md

```diff
@@ -255,7 +255,7 @@ platform. See [Customize icons topic](images.html) for more information.
    EnableWebGL(boolean) | *Default: false* <br/>  ==OS X **4.0.0**== <br/> Set to true to enable WebGL on the web view.
    ErrorUrl(URL) | *Default: null* <br/> ==Android== <br/> If set, will display the referenced page upon an error in the application instead of a dialog with the title "Application Error".
    ErrorUrl(string) | ==iOS== <br/> If set, will display the referenced local page upon an error in the application.
-   FullScreen(boolean) | *Default: false* <br/> Allows you to hide the status bar at the top of the screen.
+   FullScreen(boolean) | *Default: false* <br/> ==Android== <br/> Allows you to hide the status bar at the top of the screen. <br/> __Note__: Recommended platform-agnostic way to achieve this is to use the [StatusBar plugin](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-statusbar).
```

Comment:
It's recommended to provide a relative reference to the plugin docs as that ensures version and language is correct.